<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-V42TCMT" failures="1" tests="3" name="Default test" time="0.112" errors="0" timestamp="2025-05-08T13:04:58 IST">
  <testcase classname="automationCore.Base" name="@BeforeMethod initialiseBrowser" time="-1.746689698865E9">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;browser&amp;apos; is required by BeforeMethod on method initialiseBrowser but has not been marked @Optional or defined
in C:\Users\Ranil\AppData\Local\Temp\testng-eclipse-45594266\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'browser' is required by BeforeMethod on method initialiseBrowser but has not been marked @Optional or defined
in C:\Users\Ranil\AppData\Local\Temp\testng-eclipse-45594266\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:264)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:352)
at org.testng.internal.Parameters.createParameters(Parameters.java:690)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:163)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:313)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- @BeforeMethod initialiseBrowser -->
  <testcase classname="automationCore.Base" name="@BeforeMethod initialiseBrowser" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod initialiseBrowser -->
  <testcase classname="testScript.LoginTest" name="verifyUserLoginwithInvalidCredentials" time="0.0">
    <skipped/>
  </testcase> <!-- verifyUserLoginwithInvalidCredentials -->
  <testcase classname="testScript.LoginTest" name="verifyUserLoginwithInvalidCredentials" time="0.0">
    <skipped/>
  </testcase> <!-- verifyUserLoginwithInvalidCredentials -->
  <testcase classname="testScript.LoginTest" name="verifyUserLoginwithInvalidCredentials" time="0.071">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
[public void testScript.LoginTest.verifyUserLoginwithInvalidCredentials(java.lang.String,java.lang.String) throws java.io.IOException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserLoginwithInvalidCredentials([java.lang.String arg0, java.lang.String arg1])
Arguments: []">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void testScript.LoginTest.verifyUserLoginwithInvalidCredentials(java.lang.String,java.lang.String) throws java.io.IOException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserLoginwithInvalidCredentials([java.lang.String arg0, java.lang.String arg1])
Arguments: []
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:915)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyUserLoginwithInvalidCredentials -->
  <testcase classname="testScript.LoginTest" name="verifyUserLoginwithInvalidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- verifyUserLoginwithInvalidPassword -->
  <testcase classname="testScript.LoginTest" name="verifyUserLoginwithInvalidUsername" time="0.0">
    <ignored/>
  </testcase> <!-- verifyUserLoginwithInvalidUsername -->
  <testcase classname="testScript.LoginTest" name="verifyUserLoginwithValidCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- verifyUserLoginwithValidCredentials -->
</testsuite> <!-- Default test -->
